content/integrations/beanstalk.md

```diff
@@ -0,0 +1,89 @@
+
+A `Dockerrun.aws.json` file is an Elastic Beanstalk–specific JSON file that describes how to deploy a set of Docker containers as an Elastic Beanstalk application. You can use this file for a multicontainer Docker environment.
+
+`Dockerrun.aws.json` describes the containers to deploy to each container instance in the environment as well as the data volumes to create on the host instance for the containers to mount.
+
+A `Dockerrun.aws.json` file can be used on its own or zipped up with additional source code in a single archive. Source code that is archived with a `Dockerrun.aws.json` is deployed to container instances and accessible in the `/var/app/current/` directory. Use the volumes section of the config to provide mount points for the containers running on the instance, and the mountPoints section of the embedded container definitions to mount them from the containers.
+
+The following snippet illustrates a `Dockerrun.aws.json` declaring the Datadog agent. You can copy it, and add the definition of your other containers in it. You can then zip it with some content you want to send to the instances which run the previously described containers, and ship it to Beanstalk. For more info about the syntax of this file you can refer to [the Beanstalk documentation](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_v2config.html).
+
+~~~~~~~~
+{
+  "AWSEBDockerrunVersion": 2,
+  "volumes": [
+    {
+       "name": "docker_sock",
+       "host": {
+            "sourcePath": "/var/run/docker.sock"
+      }
+    },
+    {
+       "name": "proc_mounts",
+       "host": {
+            "sourcePath": "/proc/mounts"
+      }
+    },
+    {
+       "name": "cgroup",
+       "host": {
+            "sourcePath": "/cgroup/"
+      }
+    }
+  ],
+  "containerDefinitions": [
+    {
+      "name": "dd-agent",
+      "image": "datadog/docker-dd-agent:ecs",
+      "environment": [
+            {
+              "name": "API_KEY",
+              "value": "your_api_key"
+            },
+            {
+              "name": "TAGS",
+              "value": "simple-tag-0, tag-key-1:tag-value-1"
+            }
+      ],
+      "memory": 128,
+      "mountPoints": [
+        {
+          "sourceVolume": "docker_sock",
+          "containerPath": "/var/run/docker.sock",
+          "readOnly": false
+        },
+        {
+          "sourceVolume": "proc_mounts",
+          "containerPath": "/host/proc/mounts",
+          "readOnly": false
+        },
+        {
+          "sourceVolume": "cgroup",
+          "containerPath": "/host/sys/fs/cgroup",
+          "readOnly": false
+        }
+      ],
+    },
+  ]
+}
+~~~~~~~~
+
+#### Creating the environment
+
+Once the container definition is ready, we need to ship it to Beanstalk. [The multicontainer Docker tutorial](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_docker_ecstutorial.html) of the Elastic Beanstalk documentation explains how to do so easily.
```

Comment:
Once the container definition is set, the last step is to deploy it to Beanstalk. This step is explained in The multi...